### PR TITLE
added check if fieldLocale.class is undefined to prevent javascript error

### PR DESCRIPTION
**Pull request by ShittyCoiner**
### All Submissions:

* [ ] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [ ] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Closes # .

### How to test the changes in this Pull Request:

1.
2.
3.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully run tests with your changes locally?

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### CODE REVIEW

File: assets/js/frontend/address-i18n.js [hunk 1]
```diff
@@ -104,7 +104,9 @@ jQuery( function( $ ) {
 
 				// Class changes.
 				field.removeClass( 'form-row-first form-row-last form-row-wide' );
```
Comment (Spreeuw) [inline assets/js/frontend/address-i18n.js:106]: I would suggest to include this in the if statement too

Comment (ShittyCoiner) [reply assets/js/frontend/address-i18n.js]: I have included this line

**Comment by Spreeuw** [comment]
I'm not in the WooCommerce developer team, but I think you have a much bigger chance of this pull request being accepted if you make a bug report too that describes the issue and references this PR.

**Comment by rrennick** [comment]
@ShittyCoiner Thanks for the PR. We are currently testing #27405 with a more precise check to address this. I'll close this PR as a duplicate.

**Pull request closed without merging**